lib/policies/load-balancing.js

```diff
@@ -167,6 +169,14 @@ DCAwareRoundRobinPolicy.prototype.newQueryPlan = function (keyspace, queryOption
   var self = this;
   var counter = 0;
   var remoteCounter = 0;
+
```

Comment:
One small issue here is that we can generate some duplicate consecutive query plans if hosts is organized like:

```
aDC1, bDC2, cDC3, dDC1, eDC2, fDC3, gDC1
```

the plans returned would be:

```
aDC1, dDC1, gDC1
dDC1, gDC1, aDC1
dDC1, gDC1, aDC1
dDC1, gDC1, aDC1
gDC1, aDC1, dDC1
gDC1, aDC1, dDC1
gDC1, aDC1, dDC1
```

So it wouldn't be true round robin..will try to find a solution.